implementing-nosto/implement-search/search/README.md

```diff
@@ -166,19 +166,58 @@ You don't need to execute `api.recordSearch()`if you call `api.search(query, { t
 {% endhint %}
 
 ```javascript
-nostojs(api => {
+nostojs(async (api) => {
+
+    const searchRequest = {
+      query: "shoes",
+      products: {
+        sort: [{ field: "price", order: "asc" }],
+        filter: [{ "field": "brand", "value": "Nike" }]
+      }
+    }
+    const response = await sendGraphQlRequest(searchRequest)
```

Comment:
Shall I elaborate on this more?
I think this is something that could go wrong, the query/request part is easy